### ISSUE TITLE
use curl instead of github api, that may avoid using github token

### ISSUE THREAD

**Issue by bitsf** [aquasecurity/harbor-scanner-trivy#95]
Hi @danielpacak 
For issue #5 
I think you can consider using something like below command to get the latest release on github:
```
curl -sL -o /dev/null -w %{url_effective} https://github.com/aquasecurity/harbor-scanner-trivy/releases/latest
```

This is actually referenced from goreleaser script

**Comment by danielpacak** [comment]
> Hi @danielpacak
> For issue #5
> I think you can consider using something like below command to get the latest release on github:
> 
> ```
> curl -sL -o /dev/null -w %{url_effective} https://github.com/aquasecurity/harbor-scanner-trivy/releases/latest
> ```
> 
> This is actually referenced from goreleaser script

Thank you @bitsf for the suggestion. We'll check if we can use it with https://github.com/aquasecurity/trivy /cc @knqyf263

**Comment by bitsf** [comment]
@danielpacak Will this be fixed? harbor 2.0 will release soon.

**Comment by danielpacak** [comment]
> @danielpacak Will this be fixed? harbor 2.0 will release soon.

@bitsf We double check and unfortunately the https://github.com/aquasecurity/harbor-scanner-trivy/releases/latest endpoint is also subject to rate limit. What's more we're downloading the latest asset file for the given release. And there's also flag to download full or light DB.

So for now and Harbor 2.0 the best is to go for GitHub token if the rate limit is a concern.

**Comment by bitsf** [comment]
OK, got it.
